.github/actions/build-package/action.yml

```diff
@@ -104,7 +104,10 @@ runs:
       if: ${{ inputs.package == 'components' }}
       shell: bash
       working-directory: ${{ inputs.package }}
-      run: tar -czf ../components.tgz --strip-components=1 .
+      run: |
+        tar -czf ../components-full.tgz  .
+        tar -czf ../components.tgz --directory=lib/components .
+        tar -czf ../design-tokens.tgz --directory=lib/design-tokens .
```

Comment:
`-full` is the entire components repo, for usage in e.g. running components integration tests in a separate step.

The other two are sub-packages, as published to npm, packaged up in the 'standard' way that other published packages within our dry run are, so that they can then be used as 'local dependencies' within the demo package build.

This is the first time that we're using the individual components package sub-artifacts elsewhere in the dry run, hence needing to add this.

We could technically just use `components-full`, but that would require more components-specific logic in the 'local dependencies' script, and I'd rather try to avoid that.

Reply:
Got it - thank you for the explanation!